spec/traits.dd

```diff
@@ -442,6 +443,41 @@ tuple((Foo))
 )
 )
 
+$(SECTION2 $(GNAME getFunctionVariadicStyle),
+    $(P
+        Takes one argument which must either be a function symbol, function literal,
+        or a function pointer. It returns a string identifying the kind of
+        $(LINK2 function.html#variadic, variadic arguments) that are supported.
+    )
```

Comment:
Not sure what the implementation does, but shouldn't this trait work on function types, too?